website/docs/docs/get-started-dbt.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "dbt Quickstarts"
+id: get-started-dbt
+hide_table_of_contents: true
+pagination_next: null
+pagination_prev: null
+---
+
+Start your dbt journey by trying one of our quickstarts, which provides a step-by-step guide to help you set up dbt Cloud with a [variety of data platforms](/docs/cloud/connect-data-platform/about-connections). dbt Cloud is a scalable solution that enables you to develop, test, deploy, and explore data products using a single, fully managed software service.
+
+dbt Cloud enables teams with diverse skillsets build reliable data products, at any scale, with capabilities including:
+
+- Development experiences tailored to multiple personas (in-browser [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) or locally with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation))
+- Out-of-the-box [CI/CD workflows](/docs/deploy/ci-jobs)
+- The [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) for consistent metrics that can be delivered to any end-point
+- Domain ownership of data with multi-project [dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) setups
+- [dbt Explorer](/docs/collaborate/explore-projects) for collaborative data discovery and understanding
+
+Learn more about [dbt Cloud features](/docs/cloud/about-cloud/dbt-cloud-features) and [start your free trial](https://www.getdbt.com/signup/) today.
+
+<div className="grid--3-col">
+
+<Card
+    title="Quickstart for dbt Cloud and BigQuery"
+    body="Discover how to leverage dbt Cloud with BigQuery to streamline your analytics workflows."
```

Comment:
I understand why we might change these to avoid appearing repetitive, but for search purposes, would it help to have all of these start the same way "Learn how to..."?